tests/components/google_assistant/test_trait.py

```diff
@@ -358,46 +331,6 @@
     }
 
 
-async def test_onoff_cover(hass):
```

Comment:
Might want to keep the open & close trait test for cover if we are leaving in the code for it, which I am happy you did btw.

Reply:
No. We should remove the on/off functionality for covers completely.